inductiva/structures/plate_linear_elastic/plate_linear_elastic.py

```diff
@@ -28,13 +26,13 @@ class DeformablePlate(scenarios.Scenario):
 
     valid_simulators = [simulators.FEniCSx]
 
-    def __init__(
-        self,
-        plate: structures.plates.RectangularPlate,
-        holes_list: List[structures.holes.Hole],
-        bcs_list: List[structures.bcs.BoundaryCondition],
-        material: structures.materials.IsotropicLinearElasticMaterial,
-    ):
+    def __init__(self,
+                 plate: structures.plates.RectangularPlate,
+                 holes_list: List[structures.holes.Hole],
+                 bcs_list: List[structures.bcs.BoundaryCondition],
+                 material: structures.materials.IsotropicLinearElasticMaterial,
+                 global_refinement_factor: float = 1.0,
+                 local_refinement_factor: float = 0.0):
```

Comment:
These should be parameters of the `simulate` method and not attributes of this class. See my comment above.

Reply:
If these attributes do not belong to this class, how will the client provide them?

Reply:
As arguments to the simulate method of the scenario or the run method of the simulator.

Reply:
Make sense, I will change it, thanks